pyInterface/module/_fileManager.py

```diff
@@ -17,7 +17,7 @@ def saveFileManager(fileManagerObject, path):
 		return True
 	else:
```

Comment:
this `else` could be removed entirely, then it would be more clear that the error message and the `return False` belong together